### DISCUSSION TITLE
Any way to allow people who own private servers to have Adonis Admin?

### DISCUSSION THREAD

**Discussion by r-kinesis** [Epix-Incorporated/Adonis#467] [Q&A] [answered]
I own a roleplay group and I always leave my private servers free. Is there a way to add Adonis Admin support to people who own private servers?

**Comment by NNickey** [comment, marked as answer]
Plugin.

**Reply by r-kinesis** [reply]
Elaborate??

**Reply by Webm07** [reply]
Put this one in your plugins folder: https://github.com/Sceleratis/Adonis-Plugins/blob/master/Server/Server-GivePrivateServerOwnerAdmin.lua

**Reply by r-kinesis** [reply]
Awesome! Works like a charm! Thank you!
